docs/configuration/config-file-reference.md

```diff
@@ -3606,6 +3606,11 @@ The `limits_config` configures default and per-tenant limits imposed by Cortex s
 # CLI flag: -ingester.max-series-per-metric
 [max_series_per_metric: <int> | default = 50000]
 
+# The maximum number of active native histogram series per user, per ingester. 0
```

Comment:
Add to the description that this limit needs active series tracker to be enabled.